R/project-configuration.R

```diff
@@ -3,7 +3,7 @@
 #' @description An object storing configuration used project-wide
 #' @format NULL
 #' @export
-ProjectConfiguration <- R6::R6Class(
+ProjectConfiguration <- R6::R6Class( # nolint: object_name_linter.
```

Comment:
Bit annoying that it complains about our class names. Maybe there is an option to define a separate style for R6 class names?

Reply:
You can use these settings I setup for test in TLF package: https://github.com/Felixmil/TLF-Library/blob/gh-actions-test/.lintr

Reply:
nice